stable/enterprise/templates/analyzer_deployment.yaml

```diff
@@ -39,7 +39,11 @@ spec:
         {{- include "enterprise.common.cloudsqlContainer" . | nindent 8 }}
       {{- end }}
         - name: "{{ .Chart.Name }}-{{ $component | lower }}"
-          image: {{ .Values.image }}
+          image: {{- if and .Values.imageSpec.registry .Values.imageSpec.repository .Values.imageSpec.tag -}}
+                    {{ printf "%s/%s:%s" .Values.imageSpec.registry .Values.imageSpec.repository .Values.imageSpec.tag | trim | indent 1}}
```

Comment:
this doesn't, for example handle digests instead of a tag.

Reply:
but similar bitnami format does support digest as well in the object via the "Digest" field, so not sure if that needs to also be supported here as well.

Reply:
Good point, can certainly add it.